### PR TITLE
Add documentation for the Jellyfin media player

### PR DESCRIPTION
**Pull request by Ongy**
## Proposed change

Adds the documentation about the media player entity in the Jellyfin Integration to the documentation page.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76801/
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Just changing the target branch is not enough, to get rid of the unrelated commits you would have to rebase your branch to the `next` branch.

**Comment by frenck** [comment]
Rebased the PR with relevant commit onto the latest `next` branch

File: source/_integrations/jellyfin.markdown [hunk 1]
```diff
@@ -22,6 +22,12 @@ ha_integration_type: service
 The Jellyfin integration exposes a [Jellyfin](https://jellyfin.org/) server as a Media Source in Home Assistant.
 Support is currently limited to music and movie libraries only. Other libraries will not appear in the Media Browser. This integration has been tested with Jellyfin server version 10.6.4 and later.
 
+Additionally this integration sets up every media session connected to the Jellyfin
+server as media player in Home Assistant.
+This provides media controls for the sessions from Home Assistant.
+Browsing media inside Home Assistant in a player's context provides all libraries
+of type Movie and Series.
```
Comment (frenck) [inline source/_integrations/jellyfin.markdown]:
```suggestion
Additionally, this integration sets up every media session connected to the Jellyfin
server as a media player in Home Assistant to provide media controls for each session.

Browsing media inside Home Assistant in a player's context provides all libraries
of type Movie and Series.
```

**Review by frenck — APPROVED** [review]
Thanks, @Ongy 👍

**Pull request merged**